web/src/components/workflow-editor/edit-panel/action-edit-panel.tsx

```diff
@@ -139,7 +139,12 @@ export default function ActionEditPanel() {
 								</Text>
 								<TextArea
 									variant="surface"
-									value={actionData.args[argument.argName]}
+									value={
+										actionData.args[argument.argName] !==
+										undefined
+											? actionData.args[argument.argName]
+											: argument.defaultValue || ""
+									}
```

Comment:
Can be simplified to:

```suggestion
									value={
										actionData.args[argument.argName] || argument.defaultValue || ""
									}
```

Reply:
Just tested it. It actually doesn't work. You cannot delete or completely replace the default value. So, the approach needs to be different. You need to revert this change and look at the `buildReactFlowActionNode` function in `web/src/lib/reactflow.ts`.